### PR TITLE
Allow sorting of the Board members

### PR DESCRIPTION
**Pull request by amercader**
As requested by @liekeploeger

### CODE REVIEW

File: foundation/organisation/models.py [hunk 1]
```diff
@@ -152,11 +152,18 @@ class Board(models.Model):
     name = models.CharField(max_length=100)
     slug = models.SlugField(max_length=100, unique=True)
     description = models.TextField()
+    order = models.IntegerField(
+        blank=True, null=True,
```
Comment (serahrono) [inline foundation/organisation/models.py]: do we need the order field highlighted in brown, too?

Comment (amercader) [reply foundation/organisation/models.py]: No, I guess we won't have more than one board at the same time :) I'll remove this

**Comment by amercader** [comment]
@serahrono I've removed the order field in the Board model

File: foundation/organisation/models.py (hunk 1 above)
Comment (njxjackson87) [inline foundation/organisation/models.py]: /foundation/organisation/models.py

**Pull request merged**